vocabularies/georesources-report-types.ttl

```diff
@@ -31,6 +34,34 @@ grt:pa-22A a skos:Concept ;
     skos:prefLabel "Notice of Completion of seismic survey or scientific or technical survey"@en ;
     skos:topConceptOf <http://linked.data.gov.au/def/georesource-report> .
 
+grt:pggd-01 a skos:Concept ;
+    skos:definition "A notice to inform the Department that a petroleum well or bore will be drilled."@en ;
+    skos:inScheme <http://linked.data.gov.au/def/georesource-report> ;
+    skos:prefLabel "Notice of intention to drill a petroleum well or bore"@en ;
+    skos:altLabel "PGGD-01"@en ;

```

Comment:
Should this be notation instead of altLabel?

Reply:
```suggestion
    skos:altLabel "PGGD-01"@en ;
    skos:notation "PGGD-01" ;
```

I think this is one of those examples where it functions as both. It is the official notation and designation, but it has also become a colloquial term used by people for the form itself.